.vsts-pipelines/builds/PRBuild.yml

```diff
@@ -0,0 +1,232 @@
+phases:
+- template: ../templates/phases/default.yml
+  parameters:
+    name: 'UniverseCoherence'
+    buildArgs: >
+      $(PipebuildArgs)
+      /p:SkipTests=true
+      /p:Configuration=Release
+      /p:BuildNumber=$(Build.BuildNumber)
+
+#BEGIN SharedFx
+- template: ../templates/phases/default.yml
+  parameters:
+    name: SharedFxWin64
```

Comment:
Most of the sharedfx steps look the same. Should we make those into a template as well?